src/functions_framework/background.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2021 Google LLC
```

Comment:
Can we have more descriptive filenames?

For example:
- `event_background.py`
- `event_coversion.py`

Reply:
Done.